docs/trestle_author.md

```diff
@@ -409,6 +409,48 @@ Running `trestle author docs validate -tn docs_task -gh="Governed section"` will
 - If `--template-version 1.0.0` (`-tv`) is passed the header field `x-trestle-template-version` will be ignored and document will be forcefully validated against template of version `1.0.0`.
   Use this for testing purposes _only_ when you need to validate the document against a specific template. By default the template version will be determined based on `x-trestle-template-version` in the document.
 
+### Validating the documents against different templates
+
+Validation against multiple templates as stated before can be done, but there is another scenario that you can leverage on trestle to have more than 1 instance of a template existing in the task folder.
+
```

Comment:
"... trestle to have multiple documents in the task folder corresponding to a 1 template." or something like this.

Reply:
done